devDocs/technicalDesignOverview.md

```diff
@@ -71,9 +71,9 @@ Tasks that can be performed include moving/clicking the mouse and sending key pr
 
 #### Logging in secure mode
 `logHandler.initialize` prevents logging in secure mode.
-This is because it is a security concern to log during secure mode (e.g. passwords are logged).
-To change this for testing, patch the source build.
-`nvda.log` files are then generated in the System profile's `%TEMP%` directory.
+This is because it is a security concern to log during secure mode (e.g. passwords are logged on secure screens).
```

Comment:
Is there a definition for 'secure screens' in this doc? There should be, and we should link to it.

Reply:
While at it,, could the difference between secure mode and secure screens be documented (if not already)? I have not checked in the docs nor in the code but I think that:
* NVDA runs in secure mode when executed on secure screens (unless the debug service reg key is enabled)
* NVDA can be run in secure mode in any situation providing the `-s` flag when started; except for easier testing of secure mode, I do not know what is the use case for this flag however.

Reply:
@CyrilleB79 As you mention, easier testing is one.
Other use-cases are:
- Locked down environments, perhaps sensitive corporate systems.
- It gets used during standardized testing.

Reply:
> Other use-cases are:
> 
>     * Locked down environments, perhaps sensitive corporate systems.

Actually, that's my concern. Could a command line flag really guarantee such security? This would mean that on such locked environments, the following actions would be forbidden in order to avoid that anyone executes NVDA without the `-s` flag:
* Open Windows Explorer and click on nvda.exe
* open a console (e.g. cmd or PowerShell)
* Runs "nvda.exe" from other locations such as Windows+R dialog, Explorer address bar, task manager, etc.
* etc.

For reference, see the discussion in #10018. The final recommendation in this discussion goes to adding a new system-wide registry key for such usage.

Reply:
>Is there a definition for 'secure screens' in this doc? There should be, and we should link to it.

As secure screens and secure mode are referenced regularly in the user guide, without clear definition, I'm going to create the definitions there and reference them from the technicalDesignOverview.
I'm going to add to the technicalDesignOverview a reference to [MS Docs on UAC](https://docs.microsoft.com/en-us/windows/security/identity-protection/user-account-control/how-user-account-control-works).
I haven't been able to find any MS definitions on secure screens.

Reply:
@CyrilleB79 That seems like a valid concern, however it's a separate issue from documenting the 'intent' and how the command line args are currently used for the benefit of developers.